README.md

```diff
@@ -1,5 +1,7 @@
 # Circles
 
+You can find a live build of [Circles](https://circles.csesoc.app/degree-wizard) at [https://circles.csesoc.app/degree-wizard](https://circles.csesoc.app/degree-wizard).
```

Comment:
Could we leave out the 'degree-wizard' route and just use the base url?

Reply:
Sounds good.